src/interface.jl

```diff
@@ -725,3 +811,26 @@ In the case where `d isa Distribution`, this means
 """
 forward(d::Distribution) = forward(GLOBAL_RNG, d)
 forward(d::Distribution, num_samples::Int) = forward(GLOBAL_RNG, d, num_samples)
+
+# utility stuff
+params(td::Transformed) = params(td.dist)
```

Comment:
Is `Transformed` treelike?

Reply:
No, not yet at least. It's `params` from Distributions.jl but yeah it doesn't catch the case where the bijectors themselves are parameterized. It's not clear to me how to do this in general. There's a discussion regarding whether or not we want to depend on Flux over in Sharans PR on batch norm